netmanager/src/views/pages/Dashboard/components/Map/Map.js

```diff
@@ -1,50 +1,45 @@
-import React from "react";
-import clsx from "clsx";
-import PropTypes from "prop-types";
-import { makeStyles } from "@material-ui/styles";
-import { Map as LeafletMap, TileLayer, Popup, Marker } from "react-leaflet";
-import {
-  Card,
-  CardContent,
-  CardHeader,
-  Divider,
-} from "@material-ui/core";
-import { useEffect, useState } from "react";
-import FullscreenControl from "react-leaflet-fullscreen";
-import "react-leaflet-fullscreen/dist/styles.css";
-import L from "leaflet";
-import { GET_MONITORING_SITES_URI, GET_DATA_MAP } from "config/urls/analytics";
-import Filter from "./Filter";
-import moment from "moment-timezone";
-
+import React from 'react';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/styles';
+import { Map as LeafletMap, TileLayer, Popup, Marker } from 'react-leaflet';
+import { Card, CardContent, CardHeader, Divider } from '@material-ui/core';
+import { useEffect, useState } from 'react';
+import FullscreenControl from 'react-leaflet-fullscreen';
+import 'react-leaflet-fullscreen/dist/styles.css';
+import L from 'leaflet';
+import { GET_MONITORING_SITES_URI, GET_DATA_MAP } from 'config/urls/analytics';
+import Filter from './Filter';
+import moment from 'moment-timezone';
+import createAxiosInstance from '../../../../apis/axiosConfig';
```

Comment:
Hi @OchiengPaul442 
Thanks again!
Just wondering, do we have a way of using simple paths to minimise mistakes?

1. You could try using `Webpack Module Aliases`?
2. Or `Babel Plugin Module Resolver`?
3. Or `Create React App (CRA)`?
4. Or` ESLint "import/no-unresolved" Rule`?
5. Or `Visual Studio Code Path Intellisense`?
6. etc?